adrs/adr-002-copy-signify-ts-as-initial-commit.md

```diff
@@ -0,0 +1,27 @@
+# ADR-002 - Use Signify-TS as Basis for Initial Commit
+
+## Status
+
+Proposed
+
+## Context
+
+There are two ways to begin work on the cesr-ts project. The first is to start from scratch, pulling over relevant code from signify-ts as needed. The second is to copy the signify-ts codebase over as a whole and trimming out unrelated code.
```

Comment:
This scope can be limited to the "core" directory, it holds the primitives. You could also cross-reference with `cesride`.